File: Coding Challenges/DailyChallenge/Python/20260119_EnergyConsumption.py
def compare_energy(calories_burned, watt_hours_used):
    joules_per_calorie = 4184
    joules_per_wh = 3600

    workout_energy = calories_burned * joules_per_calorie
    device_energy = watt_hours_used * joules_per_wh

    if workout_energy > device_energy:
        return "Workout"
    elif workout_energy < device_energy:
        return "Devices"
    else:
        return "Equal"

File: Coding Challenges/Daily Challenge/2026/January/20260119-Energy-Consumption.js
function compareEnergy(caloriesBurned, wattHoursUsed) {
  const joulesPerCalorie = 4184;
  const joulesPerWh = 3600;

  const workoutEnergy = caloriesBurned * joulesPerCalorie;
  const deviceEnergy = wattHoursUsed * joulesPerWh;

  if(workoutEnergy > deviceEnergy) {
    return "Workout";
  }
  else if(workoutEnergy < deviceEnergy) {
    return "Devices";
  }
  else {
    return "Equal";
  }
}